pwndbg/commands/elf.py

```diff
@@ -16,19 +16,52 @@ def elfsections() -> None:
         elffile = ELFFile(f)
         sections = []
         for section in elffile.iter_sections():
-            start = section["sh_addr"]
-
+            start = section["sh_offset"]
+            end = start + section["sh_size"]
+            vm_start = section["sh_addr"]
+            vm_end = vm_start + section["sh_size"]
+            perm = format_section_flags(section["sh_flags"])
             # Don't print sections that aren't mapped into memory
-            if start == 0:
+            if vm_start == 0:
                 continue
 
-            size = section["sh_size"]
-            sections.append((start, start + size, section.name))
+            sections.append((section.name, start, end, vm_start, vm_end, perm))
 
         sections.sort()
 
-        for start, end, name in sections:
-            print("%#x - %#x " % (start, end), name)
+        print(
+            "%-20s %-20s %-20s %-20s %-20s %-20s"
```

Comment:
Btw we can use the `tabulate` module that is installed and used in Pwndbg for displaying column data